index.js

```diff
@@ -157,6 +157,7 @@ module.exports = function push (db, source, options, callback) {
 
     if (options.watch) {
       var queue = async.queue(function (task, done) {
+        console.log(task.type + ': ' + task.filepath)
```

Comment:
Sure I can. In that case, might as well remove all arguments from queue.push (use queue.push(true) instead)
```
queue.push({
  type: type,
  filepath: filepath,
  fileinfo: fileinfo
})
```

What would you prefer?